game/selenium_tests/base.py

```
@@ -17,19 +17,31 @@
 def log_fail(msg):
-    print(f"  {RED}❌ {msg}{RESET}")
+    try:
+        print(f"  {RED}❌ {msg}{RESET}")
+    except UnicodeEncodeError:
+        print(f"  {RED}[FAIL] {msg}{RESET}")
 
 
 def log_info(msg):
-    print(f"  {CYAN}ℹ  {msg}{RESET}")
+    try:
+        print(f"  {CYAN}ℹ  {msg}{RESET}")
```

Ruff (0.15.13)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)